sdcm/sct_events/group_common_events.py

```diff
@@ -92,6 +92,11 @@ def ignore_upgrade_schema_errors():
             db_event=DatabaseLogEvent.RUNTIME_ERROR,
             line="Could not retrieve CDC streams with timestamp",
         ))
+        stack.enter_context(DbEventsFilter(
+            db_event=DatabaseLogEvent.DATABASE_ERROR,
+            line="cql_server - exception while processing connection: seastar::nested_exception "
```

Comment:
please add some comment to explain when this one is happening ?

Reply:
that is happening during drain

Reply:
Done